server/controllers/payment.js

```diff
@@ -0,0 +1,95 @@
+require('dotenv').config();
+const Profile = require('../models/Profile');
+const asyncHandler = require('express-async-handler');
+const stripe = require('stripe')(process.env.STRIPE);
+
+exports.payPetSitter = asyncHandler(async (req, res) => {
+  let { 
+    userId, 
+    customerId, 
+    sitterId,
+    email, 
+    name, 
+    serviceId, 
+    hourlyRate, 
+    hours,
+    successUrl,
+    cancelUrl
+  } = req.body;
+
+  const orderId = req.params.id;
+
+  try {  
+    if (customerId === '' && userId) {
+      const userProfile = await Profile.findById(userId);
+      if (userProfile.customerId) {
+        res.status(400);
+        throw new Error('User has already a customer Id');
```

Comment:
A proper error description here please.

```suggestion
        throw new Error('User already has a customer Id');
```